app/views/merchants/show.html.erb

```diff
@@ -1,3 +1,13 @@
 <%= @merchant.name %>
 <%= link_to "Items Index", "/merchants/#{@merchant.id}/items" %>
-<%= link_to "Invoice Index", "/merchants/#{@merchant.id}/invoices" %>
\ No newline at end of file
+<%= link_to "Invoice Index", "/merchants/#{@merchant.id}/invoices" %>
+
+<div id="top_customers">
+<h2>Top Customers</h2>
+  <%= @merchant.top_customers.each do |customer| %>
+    <div id="customer-<%= customer.id %>">
+      <p><%= customer.first_name %> <%= customer.last_name %></p>
```

Comment:
I'm not sure what the use of the customer.full_name method is, if not to be called here. I could also see an argument for extracting the method to a view helper, but you could also argue full_name is a kind of data, instead of a kind of view/presentation logic of customer data.